tests/unit/fixtures/duplicate_functions.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# shellcheck disable=SC2317
+function func1() {
+    echo "Function 1"
+}
+
+function func2() {
+    echo "Function 2"
+}
+
+function func1() {
+    echo "Function 1 Duplicate but not test"
```

Comment:
Shouldn't this duplicated function be in the no_duplicate_functions.sh file? Otherwise, you're not checking if the functions that aren't tests can indeed be duplicated without causing errors.